test/index.js

```diff
@@ -1,7 +1,62 @@
 const test = require('tape');
-const fn = require('..');
+const Sockette = require('..');
+const MockServer = require('mock-socket').Server;
 
 test('exports', t => {
-	t.is(typeof fn, 'function', 'exports a function');
+	t.is(typeof Sockette, 'function', 'exports a function');
 	t.end();
 });
+
+test('onmessage', t => {
+	t.test('should receive data as string when server sends string', st => {
+		st.timeoutAfter(500);
+		const mockServer = new MockServer('ws://localhost:8080');
+
+		const stringFromServer = 'foo bar biz';
+
+		const ws = new Sockette('ws://localhost:8080', {
+			onmessage: e => {
+				mockServer.stop();
```

Comment:
This is clean-up that needs to be done before creating a new `MockServer`. I'd rather put it somewhere else to ensure that it is always called, but could not think of anywhere else to put it as `tape` does not have any clean-up methods.